lodepng.cpp

```diff
@@ -3460,7 +3460,7 @@ unsigned lodepng_convert(unsigned char* out, const unsigned char* in,
                          const LodePNGColorMode* mode_out, const LodePNGColorMode* mode_in,
                          unsigned w, unsigned h)
 {
-  size_t i;
+  unsigned i;
```

Comment:
This just doesn't seem right: everywhere (?) ```i``` is used in this function it is compared with size_t or passed to a function taking size_t. So there shouldn't be any compiler warnings to begin with?